src/main/java/com/olegdvd/temperature/domain/GatheredSensorData.java

```diff
@@ -1,15 +1,18 @@
 package com.olegdvd.temperature.domain;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.*;
+import java.time.Clock;
 import java.time.LocalDateTime;
-import java.util.UUID;
 
 @Entity
 public class GatheredSensorData {
```

Comment:
As it entity, and also will be stored in collections for processing, should be added hashcode-equals contract

Reply:
Implemented